### PR TITLE
Feature: User satisfaction survey plugin config

### LINKED ISSUE #7429: Help Wanted: Document missing User Satisfaction Survey plugin configuration
# Context

The Mattermost Product Documentation is missing ``config.json`` and environment variable equivalents for the following 2 [User Satisfaction Survey plugin configuration settings](https://docs.mattermost.com/configure/plugins-configuration-settings.html#user-satisfaction-surveys):
- https://docs.mattermost.com/configure/plugins-configuration-settings.html#id5
- https://docs.mattermost.com/configure/plugins-configuration-settings.html#enable-user-satisfaction-survey

# Documentation Request

Update the [User Satisfaction Survey plugin configuration settings](https://docs.mattermost.com/configure/plugins-configuration-settings.html#user-satisfaction-surveys) page to include ``config.json`` and environment variable equivalents, as detailed below.

Ensure that both the search metadata content (located directly above the section header in code) and the table-formatted content is updated as needed.

## Enable Plugin
* `config.json` path - `PluginSettings.PluginStates.com.mattermost.user-survey.Enable`
* Env variable: `N/A`

## Enable user satisfaction survey
* `config.json` path - `PluginSettings.Plugins.com.mattermost.user-survey.systemconsolesetting.EnableSurvey`
* Env variable: `N/A`

# QA Validation Criteria

In a generated docs preview:
- Searching for the 2 updated configuration settings via the Search bar return the correct ``config.json`` data.
- Visiting the docs page directly returns the correct ``config.json`` data.

### PR DESCRIPTION
**Pull request by TomerPacific**
#### Summary
This pull request includes updates to the configuration settings documentation for the Mattermost User Satisfaction Surveys plugin. The changes primarily involve updating the `config.json` paths and adding environment variable information.

#### Ticket Link
Resolves #7429 

screen

### CODE REVIEW

File: source/configure/plugins-configuration-settings.rst [hunk 1]
```diff
@@ -1347,15 +1347,16 @@ Access the following configuration settings in the System Console by going to **
...
-| - **true**: (Default) Enables the Mattermost User Satisfaction Surveys plugin on your Mattermost workspace.   | - System Config path: **Plugins > User Satisfaction Surveys** |
-| - **false**: Disables the Mattermost User Satisfaction Surveys plugin on your Mattermost workspace.           |                                                               |
-+---------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+| - **true**: (Default) Enables the Mattermost User Satisfaction Surveys plugin on your Mattermost workspace.   | - System Config path: **Plugins > User Satisfaction Surveys**                                |
+| - **false**: Disables the Mattermost User Satisfaction Surveys plugin on your Mattermost workspace.           | - ``config.json`` setting: ``PluginSettings.PluginStates.com.mattermost.user-survey.Enable`` |
```
Comment (cwarnermm) [inline source/configure/plugins-configuration-settings.rst:1352]: Please update the ``config.json`` value in the table.

File: source/configure/plugins-configuration-settings.rst [hunk 2]
```diff
@@ -1364,12 +1365,13 @@ Enable plugin
...
-| **Note**: See the `Mattermost Privacy Policy <https://mattermost.com/privacy-policy/>`__ for more information on the collection and use of information by Mattermost.                       |
-+-----------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------+
++-----------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------+
+| - **true**: A survey is sent to all users every quarter. Results are used by Mattermost, Inc. to improve the product.       | - System Config path: **Plugins > User Satisfaction Surveys**                                                      |
+| - **false**: (Default) User satisfaction surveys are disabled.                                                              | - ``config.json`` setting: ``PluginSettings.Plugins.com.mattermost.user-survey.systemconsolesetting.EnableSurvey`` |
```
Comment (cwarnermm) [inline source/configure/plugins-configuration-settings.rst:1370]: Please update the ``config.json`` value in the table.

**Review by cwarnermm — APPROVED** [review]
Thank you, @TomerPacific!

**Comment by lindy65** [comment]
Assigned to you for testing @Rutam21 

You can test the changes using the preview environment [here](https://github.com/mattermost/docs/pull/7445#issuecomment-2402362130)

**Comment by Rutam21** [comment]
### The changes through this PR have been tested in the preview environment and work as expected.

Additional information on the `config.json` setting and `Environment variable` has been added in the User Satisfaction Surveys section under the **Enable plugin** and **Enable user satisfaction survey** sub-sections. I've attached a video as a confirmation.

https://github.com/user-attachments/assets/34be034d-3bf2-439a-becc-978434036a73

cc: @lindy65

**Comment by lindy65** [comment]
Thank you @Rutam21 - counted towards Hacktoberfest!

cc @cwarnermm

**Pull request merged**